importScripts/sampleRecords/library.source.json

```diff
@@ -0,0 +1,738 @@
+  "WherePublished": "Perth",
+  "Subject": "Batavia (Ship); Shipwrecks",
+  "CurrentLocation": "Large book shelves",
+  "Illustrations": "1",
+  "Maps": "1",
+  "Marks": "332p. Illus., maps, photos",
+  "Pages": "332",
+  "Rare": "0",
+  "Restrictions": "NOT FOR LOAN",
+  "DateDue": "2002-07-12",
+  "DateOut": "2002-06-14",
+  "Borrower": "NOT FOR LOAN",
+  "Value": "0"
+ },
+ {
+  "PublicationID": "120",
+  "Title": "National estate values in the Southern Forest region of South-West Western Australia",
+  "LegacyID": "120",
+  "Series": "0",
+  "Size": "Large",
+  "Authors": "Australian Heritage Commission ;  Dept of CALM",
+  "LastName": "Australian Heritage Commission & Dept of CALM",
+  "HomeLocation": "LIBRARY",
+  "LastEditBy": "LIBRARY",
+  "LastEditDate": "2013-02-19 00:00:00",
+  "LibraryNo": "Q  AUS",
+  "PublicationType": "Book",
+  "PublicationYear": "1992",
+  "Subject": "Heritage sites; Forests and forestry",
+  "CurrentLocation": "LARGE BOOK SECTION",
+  "Illustrations": "0",
+  "Maps": "1",
+  "Marks": "5 vols. Maps, tables.",
+  "Pages": "0",
+  "Rare": "0",
+  "Restrictions": "NOT FOR LOAN",
+  "DateDue": "2002-02-21",
+  "DateOut": "2002-01-24",
+  "VolumeNum": "5 vols.",
+  "Borrower": "NOT FOR LOAN",
+  "Value": "0"
+ },
+ {
+  "PublicationID": "1239",
+  "Title": "Nyungar - the people, aboriginal customs in the southwest of Australia",
+  "LegacyID": "1239",
+  "Series": "0",
+  "Authors": "Green, Neville",
+  "LastName": "Green",
+  "FirstNames": "Neville",
+  "HomeLocation": "LIBRARY",
+  "LastEditBy": "TrishA",
+  "LastEditDate": "2002-06-29 00:00:00",
+  "LibraryNo": "GRE",
+  "PublicationType": "Book",
+  "PublicationYear": "1979",
+  "Publisher": "Creative Creative Research",
+  "WherePublished": "Perth",
+  "Subject": "Nyungar (Australian people); Aborigines - Albany region -  Language -  culture;",
+  "Illustrations": "1",
+  "Maps": "1",
+  "Marks": "206p. Illus., maps, photos",
+  "Pages": "206",
+  "Rare": "0",
+  "Restrictions": "NOT FOR LOAN",
+  "DateDue": "2002-07-27",
+  "DateOut": "2002-06-29",
+  "Borrower": "NOT FOR LOAN",
+  "Value": "0"
+ },
+ {
+  "PublicationID": "12566",
+  "Title": "Mission cottage restoration (New Norcia Studies, No.17, 2009, pp.46-54)",
+  "LegacyID": "12566",
+  "Series": "0",
+  "Authors": "Bremen, Ingrid H van; Campbell, Robin McK",
+  "LastName": "Bremen",
+  "FirstNames": "Ingrid H van",
+  "OtherAuthors": "Campbell, Robin McK",
+  "HomeLocation": "LIBRARY",
+  "LastEditBy": "JENM",
+  "LastEditDate": "2015-03-19 00:00:00",
+  "LibraryNo": "J   NEW",
+  "PublicationType": "Journal article",
+  "PublicationYear": "2009",
+  "Publisher": "Benedictine Community of New Norcia",
+  "WherePublished": "New Norcia",
+  "Subject": "Cottages - New Norcia - Conservation and restoration",
+  "CurrentLocation": "ANNEXE",
+  "Illustrations": "1",
+  "Maps": "0",
+  "Pages": "0",
+  "Rare": "0",
+  "Restrictions": "NOT  FOR  LOAN",
+  "DateDue": "2011-06-02",
+  "DeweyNumber": "No. 17",
```

Comment:
A single record with a value, and this isn't on the new library form so this value is not migrated. Clarifying with client.